index.js

```diff
@@ -2149,7 +2343,35 @@
   //. false
   //. ```
   Either.prototype['fantasy-land/equals'] = function(other) {
-    return other.isLeft === this.isLeft && Z.equals(other.value, this.value);
+    return this.isLeft === other.isLeft && Z.equals(this.value, other.value);
+  };
+
+  //# Either#fantasy-land/lte :: (Ord a, Ord b) => Either a b ~> Either a b -> Boolean
+  //.
+  //. Takes a value `e` of the same type and returns `true` if:
+  //.
+  //.   - `this` is a Left and `e` is a Right; or
+  //.
+  //.   - `this` and `e` are both Lefts or both Rights, and the value of `this`
+  //.     is less than or equal to the value of `e` according to [`lte`](#lte).
+  //.
+  //. ```javascript
+  //. > S.lte_(S.Left(10), S.Right(0))
+  //. true
+  //.
+  //. > S.lte_(S.Right(0), S.Left(10))
+  //. false
+  //.
+  //. > S.lte_(S.Right(0), S.Right(1))
+  //. true
+  //.
+  //. > S.lte_(S.Right(1), S.Right(0))
+  //. false
+  //. ```
+  Either.prototype['fantasy-land/lte'] = function(other) {
```

Comment:
Shouldn't this be implemented conditionally, similarly to how we did Semigroup on Maybe? Or do all JavaScript values have `Ord`, as is the case with `Setoid`?